avae/config.py

```diff
@@ -131,8 +131,12 @@ class AffinityConfig(BaseModel):
     vis_los: bool = Field(False, description="Visualise loss")
     vis_pos: bool = Field(False, description="Visualise pose")
     vis_pose_class: str = Field(
-        None, description="Visualise pose classification"
+        None, description="Visualise pose per class interpolation"
     )
+    vis_z_n_int: str = Field(
```

Comment:
i think here default should be none, otherwise it will fail in a different dataset if variable is not declared

Reply:
oh right, I misunderstood, this is ok!